RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -297,29 +257,11 @@ void ElectronSeedProducer::filterClusters(const reco::BeamSpot& bs,
         hoe2s.push_back(std::numeric_limits<float>::infinity());
       }
     }
-
-    if (applySigmaIEtaIEtaCut_ == true) {
-      noZS::EcalClusterLazyTools lazyTool_noZS(event, setup, ebRecHitCollection_, eeRecHitCollection_);
-      std::vector<float> vCov = lazyTool_noZS.localCovariances(*(scl.seed()));
-      int detector = scl.seed()->hitsAndFractions()[0].first.subdetId();
-      if (detector == EcalBarrel)
-        sigmaIEtaIEtaEB_.push_back(edm::isNotFinite(vCov[0]) ? 0. : sqrt(vCov[0]));
-      if (detector == EcalEndcap)
-        sigmaIEtaIEtaEE_.push_back(edm::isNotFinite(vCov[0]) ? 0. : sqrt(vCov[0]));
-    }
```

Comment:
This part that is executed when `applySigmaIEtaIEtaCut_ == true` got removed because it does nothing apart from filling the `sigmaIEtaIEtaEB_` and `sigmaIEtaIEtaEE_` vectors which have never been used elsewhere. As a consequence, the ElectronSeedProducer could be slimmed down also in other spots.